opm/input/eclipse/Parser/Parser.cpp

```diff
@@ -708,8 +708,10 @@ std::optional<std::filesystem::path> ParserState::getIncludeFilePath( std::strin
         std::string stringStartingAtPathName = path.substr(positionOfPathName+1);
         size_t cutOffPosition = stringStartingAtPathName.find_first_not_of(validPathNameCharacters);
         std::string stringToFind = stringStartingAtPathName.substr(0, cutOffPosition);
-        std::string stringToReplace = this->pathMap.at( stringToFind );
-        replaceAll(path, pathKeywordPrefix + stringToFind, stringToReplace);
+
+        if (auto candidate = this->pathMap.find(stringToFind); candidate != this->pathMap.end()) {
+           replaceAll(path, pathKeywordPrefix + stringToFind, candidate->second);
+        }
```

Comment:
If `candidate == end` here, then the input is malformed and we should abort processing.

Reply:
We will keep a local patch in ResInsight for this change